testsuite/views.py

```diff
@@ -76,6 +78,23 @@ def authorized():
     exceptions.abort(500)
 
 
+@app.route('/health_summary/')
+def health_summary():
+
+    latest_results = {}
+    vendors = get_names()
+
+    for vendor in vendors:
+        last_test = TestRun.query.filter_by(vendor=vendor)\
+            .order_by(desc(TestRun.date_ran)).first()
```

Comment:
StackOverflow documentation for `TestRun.date_ran.desc()` - https://stackoverflow.com/a/4187279